### PR TITLE
The Two Sum GOD is back

### PR DESCRIPTION
**Pull request by Brady-Malott**

### CODE REVIEW

File: brady/valid_anagram.java
Comment (zainaraza43) [inline brady/valid_anagram.java:1]: This file shouldn't be getting added a second time.. Fix please.

**Review by zainaraza43 — CHANGES_REQUESTED** [review]
You aren't committing `valid_anagram.java` a second time, fix your PR.

File: brady/valid_anagram.java
Comment (WahidBawa) [reply brady/valid_anagram.java:1]: L + ratio

**Review by zainaraza43 — APPROVED** [review]
LGTM :rocket:

**Pull request merged**